R/importNet.Multinet.R

```diff
@@ -0,0 +1,145 @@
+# importNet.MULTINET.R
+
+#' Import network data from BIOGRID, ENCODE data, KEGG, and SignaLink database 
+#' (Multinet interactions).
+#'
+#' \code{importNet.MULTINET} imports network edges from a MULTINET database file,
+#' 
+#' @section Networks:
+#'   MULTINET "protein.links.detailed.v10.txt" files contain
+#'   several protein networks: neighborhood, fusion, cooccurence, coexpression,
+#'   experimental, database, textmining, and combined_score. However this
+#'   function is not restricted to these types, but will read one network for
+#'   which the column name is requested in the function's net parameter. This
+#'   allows users to define their own column.
+#' @section The gG object:
+#'   The function returns a rete gG object, a weighted,
+#'   directed, simple igraph graph in which HGNC gene symbols are vertex names,
+#'   and the edge attributes $weight hold the network scores. Graph attributes
+#'   hold metadata; use igraph::graph_attr(gG) to return: $version: the gG
+#'   object version; $logFile: the filename to which log information was
+#'   written; $inFile: the input filename of MULTINET data; $call: the complete
+#'   function call with expanded arguments; and $date: when the gG object was
+#'   created.
+#'
+#' @param fName Filename of a MultiNet Multinet.interactions.network_presence.txt file.
+#' @param net The requested network. This must be a MULTINET that exists in the
+#'   header. The default is "PPI".
+#' @param verbose Controls whether to print summary information after import. 
```

Comment:
I don't know where this is coming from - but there is no behaviour in the function that is controlled by this flag. Remove it!